app/views/filter/_filter_content.html.erb

```diff
@@ -52,7 +60,7 @@
   <div class="filter__item <%= "filter__item--active" if params[:author] %> dropdown" data-dropdown data-filter data-default="Author...">
     <%= link_to params[:author] ? params[:author].split("#")[0] : "Author...", "#",
                 data: { action: "toggle-dropdown get-verified-users", filter_type: "author", value: (CGI.escape(params[:author]) if params[:author].present?) } %>
-    <%= link_to "X", "#", class: "filter__remove", data: { action: "add-filter", value: "" } %>
+    <%= link_to "X", "#", class: "filter__remove", data: { action: direct ? nil : "add-filter", value: "" } %>
```

Comment:
```suggestion
    <%= link_to "X", build_filter_path(:author, nil), class: "filter__remove", data: { action: direct ? nil : "add-filter", value: "" } %>
```

Reply:
With this, I noticed selecting an author removes the search term. Is that intentional?

Reply:
Nope!